ci-operator/step-registry/ipi/conf/ibmcloud/ipi-conf-ibmcloud-chain.yaml

```diff
@@ -0,0 +1,14 @@
+chain:
+  as: ipi-conf-ibmcloud
+  steps:
+  - ref: ipi-conf
+  - ref: ipi-conf-ibmcloud
+  - ref: ipi-conf-manual-creds
```

Comment:
That step only adds `credentialsMode: Manual` to the `install-config.yaml`. Why not just hardcode it here?

Reply:
Yes, I expect we'll add a new step after this to perform any configuration we need for supporting the CI, similar to what AWS does, as a new step to be added once those details are worked out.
https://github.com/openshift/release/blob/7e61829f682e1574513f78c3e94537836d824ab3/ci-operator/step-registry/ipi/aws/pre/manual-oidc-sts/ipi-aws-pre-manual-oidc-sts-chain.yaml#L6